libkbfs/choosedefaultmount_windows.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// +build windows
+
+package libkbfs
+
+import (
+	"errors"
+	"strings"
+)
+
+func chooseDefaultMount(dirs []string) (string, error) {
+	var dir string
+	if len(dirs) == 0 {
+		return "K:", errors.New("chooseDefaultMount fails - nothing to choose from")
```

Comment:
I'm fine with using "K:" as a default, but in that case the return error should be `nil`, or the function shouldn't return an error at all.  It's bad Go to return a usable value along with an error.